lblod/frontend-lokaal-mandatenbeheer
# reactivate rangorde for gemeenteraadsleden

## Description

Adds rangorde back for gemeenteraadsleden.

In prepare legislatuur sectie (rangorde is generated when generating rows):
![image](https://github.com/user-attachments/assets/f84606e9-4215-481e-b26d-6fa9f3fabbd6)
In create mandataris form when selecting gemeenteraadslid:
![image](https://github.com/user-attachments/assets/fed29131-9bee-45cb-9a53-d7b5fa6afc9c)
In edit mandataris form when selecting gemeenteraadslid:
![image](https://github.com/user-attachments/assets/1cbc1281-01ec-4674-9ac9-d687e1109acb)
But not others (except schepenen):
![image](https://github.com/user-attachments/assets/f404590c-36ee-48ea-a61d-39c432504dfa)
Rangode is shown in gemeenteraadslid detail page
![image](https://github.com/user-attachments/assets/7cd925cf-a59f-4362-8f4b-1c780334b592)


## How to test

- Generate some rows for gemeenteraadsleden in the prepare legislatuur section, see the rangorde appear
- notice that this does not happen for e.g. bcsd
- Add a gemeenteraadslid in prepare legislatuur, see you can add a rangorde
- edit a gemeenteraadslid in prepare legislatuur, see you can add a rangorde
- view the details of a gemeenteraadslid of previous governing periods, see you can view and edit rangorde
